First draft of updated AOP content (#1202)
cloudflare/cloudflare-docs: products/ssl/src/content/origin-configuration/authenticated-origin-pull.md

# Authenticated origin pull

Authenticated origin pulls add an extra layer of security to your domain. 

> These authenticated pulls become particularly important when you start using the Cloudflare Web Application Firewall (WAF), because you can make sure that **all traffic** is evaluated before reaching your origin server.

Reviewer: This sentence is a bit hard to read. I think it would be better to break it up. I don't know if it's accurate to say it this way, but maybe something like 
```suggestion
This authentication is important in use with the Cloudflare Web Application Firewall (WAF). In conjunction with the Cloudflare WAF, you can evaluate **all traffic** before it reaches your origin.
```

<ButtonGroup>
    <Button type="primary" href="#set-up-authenticated-origin-pulls">Get started</Button>
    <Button type="secondary" href="#how-authenticated-origin-pulls-work">Learn more</Button>
</ButtonGroup>

<Aside type='warning' header='Important'>

Authenticated Origin Pull is incompatible with Railgun.

</Aside>

## How authenticated origin pulls work

### Simple explanation

When visitors request content from your domain, Cloudflare first attempts to serve content from the cache. Failing that, Cloudflare sends a request — or an `origin pull` — back to your origin web server to get the content.

> Authenticated origin pulls make sure that all of these `origin pulls` come from Cloudflare. Put another way, authenticated origin pulls ensure that any HTTPS requests outside of Cloudflare are blocked from reaching your origin. It's a great way to enhance security.

Reviewer: I'm not sure the last sentence here on enhancing security is necessary since it's mentioned above in the first sentence on the page.

Reviewer: I love this simple explanation section! It makes it easier to understand for those that don't want/need the technical details
